app/lib/actions/katello/host/register.rb

```diff
@@ -99,8 +99,10 @@ def plan_system(system, content_view_environment, consumer_params)
 
         def plan_content_facet(host, content_view_environment)
           content_facet = host.content_facet || ::Katello::Host::ContentFacet.new
```

Comment:
Should `ContentFacet.new` just be passed the host object instead of assigning it below?